service/src/main/kotlin/fi/espoo/evaka/children/ChildrenQueries.kt

```diff
@@ -64,3 +65,19 @@ SELECT child_id FROM foster_parent WHERE parent_id = ${bind(userId)} AND valid_d
             )
         }
         .toList()
+
+fun Database.Read.getActivePlacementUnitsForChildren(
+    today: LocalDate,
+    childIds: Set<ChildId>
+): Map<DaycareId, List<ChildId>> =
+    createQuery {
+            sql(
+                """
+SELECT unit_id, child_id
+FROM placement
+WHERE child_id = ANY (${bind(childIds)}) AND start_date <= ${bind(today)} AND end_date >= ${bind(today)}
```

Comment:
I think range operations are usually preferred
`AND daterange(start_date, end_date, '[]') @> ${bind(today)}`